sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -370,6 +370,8 @@ abstract class JdbcDialect extends Serializable with Logging{
    */
   def supportsLimit(): Boolean = true
 
+  def supportsTopN(): Boolean = true
```

Comment:
is there any database that do not support limit and sort?

Reply:
I guess not exists the one.